kayvee.go

```diff
@@ -4,6 +4,14 @@ import (
 	"encoding/json"
 )
 
+// Log Levels:
+const Unknown = "unknown"
```

Comment:
I think adding the type is a good idea. It forces people to use the right type instead of accidentally typing in a string. It's a breaking change but won't affect people until we create a new tag and they import that instead of v0